components/clm/src/biogeochem/PhenologyMod.F90

```diff
@@ -967,8 +968,9 @@ subroutine CNStressDecidPhenology (num_soilp, filter_soilp , &
          
          soilpsi                             =>    soilstate_vars%soilpsi_col                            , & ! Input:  [real(r8)  (:,:) ]  soil water potential in each soil layer (MPa)   
          
-         t_soisno                            =>    col_es%t_soisno                         , & ! Input:  [real(r8)  (:,:) ]  soil temperature (Kelvin)  (-nlevsno+1:nlevgrnd)
-         
+         t_soisno                            =>    col_es%t_soisno                                       , & ! Input:  [real(r8)  (:,:) ]  soil temperature (Kelvin)  (-nlevsno+1:nlevgrnd)
+         h2osfc                              =>    col_ws%h2osfc                                         , & ! Input:  [real(r8) (:)   ]  surface water (mm)
+         salinity                            =>    col_ws%salinity                                        , & ! Input:  [real(r8) (:)   ]  salinity (TAO 5/19/2020)
```

Comment:
this salinity component is finished yet.  Its a placeholder for when I finish the ELM-PFLOTRAN updates